pom.xml

```diff
@@ -38,7 +38,7 @@
     <cobertura.version>2.0.3</cobertura.version>
     <cobertura.maven.plugin.version>2.6</cobertura.maven.plugin.version>
     <failsafe.maven.plugin.version>2.4.3-alpha-1</failsafe.maven.plugin.version>
-    <findbugs.maven.plugin.version>2.5.3</findbugs.maven.plugin.version>    
+    <findbugs.maven.plugin.version>3.0.4</findbugs.maven.plugin.version>    
     <hotels.oss.plugin.config.version>1.0.4</hotels.oss.plugin.config.version>
     <jdk.version>1.7</jdk.version>
```

Comment:
ok I thought you had a project building to java 1.8. thanks sounds good and I agree let's not bump up the version in that case. approving...